index.html

```diff
@@ -4975,6 +4975,621 @@ <h2 id="AudioWorkerParamDescriptor">
+                  can contain multiple channels.
+                </dd>
+                <dt>
+                  Object parameters
+                </dt>
+                <dd>
+                  Represents an object contains multiple arrays of
+                  <a>AudioParam</a> data with the associated name as a key.
+                </dd>
+              </dl>
+            </dd>
+            <dt>
+              void postMessage()
+            </dt>
+            <dd>
+              <p>
+                It may be called to send a message to the
+                <a>AudioWorkletNode</a>, via the algorithm defined by the
+                Worker specification.
+              </p>
+              <dl class="parameters">
+                <dt>
+                  any message
+                </dt>
+                <dd>
+                </dd>
+                <dt>
+                  optional sequence&lt;Transferable&gt; transfer
+                </dt>
+                <dd>
+                </dd>
+              </dl>
+            </dd>
+          </dl>
+          <section>
+            <h2 id="AudioParamDescriptor">
+              AudioParamDescriptor
+            </h2>
+            <p>
+              The AudioParamDescriptor dictionary is used to specify properties
+              for an <a>AudioParam</a> object that is used in an
+              <a>AudioWorkletNode</a>.
+            </p>
+            <dl title="dictionary AudioParamDescriptor" class="idl">
+              <dt>
+                DOMString name
+              </dt>
+              <dd>
+                Represents the name of a parameter. An exception will be thrown
+                when the duplicated name is found when registering the class
+                definition.
+              </dd>
+              <dt>
+                float defaultValue = 0
+              </dt>
+              <dd>
+                Represents the default value of the parameter. If this value is
+                out of the range of float data type or the range defined by
+                minValue and maxValue, an exception will be thrown.
+              </dd>
+              <dt>
+                float minValue
+              </dt>
+              <dd>
+                Represents the minimum value. An exception will be thrown, if
+                this value is out of range of float data type or it is greater
+                than maxValue.
+              </dd>
+              <dt>
+                float maxValue
+              </dt>
+              <dd>
+                Represents the maximum value. An exception will be thrown, if
+                this value is out of range of float data type or it is smaller
+                than minValue.
+              </dd>
+            </dl>
+          </section>
+
+          <section>
+            <h2 id="AudioContextInfo">
+              AudioContextInfo
+            </h2>
+            <p>
+              The AudioContextInfo dictionary is used to query the current
```

Comment:
No. The dictionary is used to return the information from a query.  Rephrase this somehow.